service/src/test/java/io/camunda/service/UserServiceTest.java

```diff
@@ -76,15 +76,16 @@ public void shouldTriggerDeleteRequest() {
     // when
     when(authentication.claims()).thenReturn(Map.of());
     final var userRecord = new UserRecord();
-    userRecord.setUserKey(1234L);
+    final var username = "test";
+    userRecord.setUserKey(1234L).setUsername(username);
```

Comment:
❓ Do we still need to set the user key here?

Reply:
Not really, no. I can remove it.